definitions/npm/styled-components_v4.x.x/flow_v0.104.x-/styled-components_v4.x.x.js

```diff
@@ -245,44 +251,60 @@ declare module 'styled-components' {
 
   declare export function withTheme<Theme, Config: {...}, Instance>(Component: React$AbstractComponent<Config, Instance>): React$AbstractComponent<$Diff<Config, ThemeProps<Theme | void>>, Instance>
 
-  declare export type StyledComponent<Props, Theme, Instance> = React$AbstractComponent<Props, Instance> & InterpolatableComponent<Props>
-  declare interface StyledFactory<StyleProps, Props, Theme, Instance> {
+  // NOTE: I removed interpolable component because it breaks jsx instantiation
+  declare export type StyledComponent<Props, Theme, Instance> = React$AbstractComponent<Props, Instance> & InterpolatableComponent<any>
+
+  declare type StyledFactory<StyleProps, Props, Theme, Instance> = {
```

Comment:
Also, as mentioned in https://github.com/flow-typed/flow-typed/pull/3228#issuecomment-522264550, I think `Props` can go away.